docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,120 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handle Application Control Actions
```

Comment:
@hjhun Also kindly change 'Handle' to 'Handling' here, as in bullet points above we are mentioning as "Handling" only.